firebase-common/src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -668,4 +675,13 @@ public void execute(@NonNull Runnable command) {
       HANDLER.post(command);
     }
   }
+
+  @Nullable
+  private static String detectKotlinVersion() {
+    try {
+      return kotlin.KotlinVersion.CURRENT.toString();
+    } catch (NoClassDefFoundError ex) {
```

Comment:
Based on cursory research, this seems like the only exception that could be caused.
Are there other exceptions we should be watching out for ? ClassNotFoundException ?

Reply:
Nope, this is the only one that can be thrown